tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -278,6 +302,18 @@ def parse_args():
         action="store_true",
         help="Use LLMPipeline from transformers library to instantiate the model.",
     )
+    parser.add_argument(
+        "--image-size",
```

Comment:
what do you think about addition seed as argument for random generation stability?

Reply:
thanks, now the seed is fixed and I was planning to expose it in the follow-up PRs but can do it now